SingleSource/UnitTests/AArch64/acle-fmv-features.c

```diff
@@ -36,58 +37,59 @@ static bool any_fails = false;
     static void check_##FN_NAME_SUFFIX(void) { \
         printf("%s\n", #FMV_FEATURE); \
         fflush(stdout); \
-        if (safe_try_feature(try_##FN_NAME_SUFFIX)) { \
+        /* default versions are allowed to UPASS when IS_EXEMPT = true */ \
+        if (safe_try_feature(try_##FN_NAME_SUFFIX, #IS_EXEMPT)) { \
```

Comment:
```suggestion
        if (safe_try_feature(try_##FN_NAME_SUFFIX, IS_EXEMPT)) { \
```
Should not stringify here.